src/library/treeitem.cpp

```diff
@@ -84,7 +84,6 @@ void TreeItem::appendChild(TreeItem* pChild) {
 
 void TreeItem::insertChild(TreeItem* pChild, int row) {
 	DEBUG_ASSERT(row >= 0);
-	DEBUG_ASSERT(row < m_children.size());
```

Comment:
This isn't related to the std::move() issue right? Is it a mistake? If not, it should really be in a separate commit